package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "telemetry-events",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
To me this is either a patch or a declaration of es6 compatibility. And...the README wasn't updated to reflect that. ...so right now this seems to be just a patch.